invenio_app_ils/ui/backoffice/src/pages/UserDetails/components/PatronLoansTable/PatronLoansTable.js

```diff
@@ -0,0 +1,245 @@
+import React, { Component } from 'react';
```

Comment:
I am sorry, I am a bit confused. It looks that this component would like to be generic (accepting an optional `patron` or `loanState` as props for example), but in reality it is very specific, it is for patron loans and cannot be reused because it is connected to a specific part of the state.

I think that what we wanted to do was to have a generic loans component for presentation only, but the wrapper is specific. For example:
```
<PatronLoans>
    <LoansTable
        headers={[]}
        loans={[]}
        onSortByChangeHandler={func}
        onSortOrderChangeHandler={func}
    />
<PatronLoans>
```

If we don't what to do it now, then there is no point to have this optional props.